packages/core/src/types/IDIDManager.ts

```diff
@@ -124,6 +124,28 @@ export interface IDIDManagerGetOrCreateArgs {
   options?: object
 }
 
+export interface IDIDManagerUpdateArgs {
+  /**
+   * Required. DID
+   */
+  did: string
+
+  /**
+   * Identifier alias. Can be used to reference an object in an external system
+   */
+  alias?: string
+
+  /**
+   * Optional. Key Management System
+   */
+  kms?: string
```

Comment:
I think this is not really needed here since the plugin methods get access to `context` which should allow them to call `IKeyManager` methods.
Specifying a particular `kms` here suggests that the `DIDManager` plugin should also know how to work with key management systems directly, which is not the case.

I suppose this would be needed if during the update, this agent should also be importing keys into a particular KMS. I prefer to keep these 2 actions separate.

Reply:
Good point!
This can be handled by `Import` methods, either `didManagerImport` or `keyManagerImport`.